### ISSUE TITLE
get Nan!

### ISSUE THREAD

**Issue by Khoa-NT** [Totoro97/f2-nerf#30]
Hi @Totoro97,

Thank you for your great work.
I tried to train your code with my own dataset.

> python scripts/run.py --config-name=wanjinyou \
> dataset_name=<your-dataset-name> case_name=<your-case-name> mode=train \
> +work_dir=$(pwd)

But I got `Nan!`. 
Can you help me? How to modify the configuration in `wanjinyou` to overcome the `Nan`?

Thank you

**Comment by Totoro97** [comment]
Hi, how is the synthesized image quality? It is OK to see one or two `Nan`s in the console. This printing message means we find nan numbers from backward gradients and we will skip this iteration and not optimize the parameters in this training step.

**Comment by Khoa-NT** [comment]
Hi, thank you for reaching out.

> how is the synthesized image quality?

I trained with real photos which are photos of my campus building.
This is the log 
[train_log.txt](https://github.com/Totoro97/f2-nerf/files/11436856/train_log.txt)
In the end, the loss is always `Nan`.
